src/mscorlib/src/System/IO/StreamReader.cs

```diff
@@ -71,6 +71,9 @@ internal static int DefaultBufferSize
         // This is used only for preamble detection
         private int bytePos;
 
+        [NonSerialized]
+        private StringBuilder builder;
```

Comment:
New private fields should be named with an underscore. i.e. `_builder`
(Existing fields can't be renamed for compat reasons. i.e. reflection)